### PR TITLE
My reviews

### PR DESCRIPTION
**Pull request by Kwolik**
- Create page my reviews
- Create file MyReviewsOneReview.jsx , MyReviewsCategories.jsx

image

### CODE REVIEW

File: web-client/src/components/MyReviewsCategories.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            textTransform: 'none',
+        }
+    });
+
+    const style = styles();
+
+    return(
+        <div className={style.main}>
+            <Button variant='outlined' color="primary" 
+                            className={style.btn}>{props.name}</Button>
```
Comment (KamilDonda) [inline web-client/src/components/MyReviewsCategories.jsx:25]: Move </Button> lower, please :D

**Review by KamilDonda — APPROVED** [review]

File: web-client/src/components/MyReviewsOneReview.jsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        btn2:
+        {
+            float: 'left',
+            textTransform: 'none',
+        },
+    });
+
+    const style = styles();
+
+    //https://www.xspdf.com/resolution/50694881.html <- informacje do buttona download
```
Comment (JustD4nTe) [inline web-client/src/components/MyReviewsOneReview.jsx]: eng pls

File: web-client/src/components/MyReviewsOneReview.jsx [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+            float: 'left',
+            textAlign: 'left',
+            margin: 'auto',
+            marginTop: '1%',
+        },
+        photo:
+        {
+            float: 'left',
+            width: '6%',
+            height: '35px',
```
Comment (JustD4nTe) [inline web-client/src/components/MyReviewsOneReview.jsx:36]: Width depends on size of website, but height have a constant value. @upsu90, he should leave it for now?
image
image

**Review by JustD4nTe — CHANGES_REQUESTED** [review]

File: web-client/src/components/MyReviewsOneReview.jsx (hunk 3 above)
Comment (Kwolik) [reply web-client/src/components/MyReviewsOneReview.jsx:36]: I did that all the time because height I can not describe in %

Comment (JustD4nTe) [reply web-client/src/components/MyReviewsOneReview.jsx:36]: I didn't notice this earlier :/ The best solution is min/max properties, I guess 😄 So, you can leave it. Later we'll change it

Comment (Kwolik) [reply web-client/src/components/MyReviewsOneReview.jsx:36]: it's best to leave it and then see how these avatars scale

Comment (upsu90) [reply web-client/src/components/MyReviewsOneReview.jsx:36]: I think it'll be better to change it now, we never want to change the aspect ratio

Comment (upsu90) [reply web-client/src/components/MyReviewsOneReview.jsx:36]: Maybe you can set one value and the other will be "auto"

File: web-client/src/pages/my-reviews.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+            title: 'Importance of Golden Ratio in Mathematics',
+            categories: 'Mathematics',
+            data: '30/11/2020',
+            authors: 'John Doe, Sam Smith, Ashley Blue',
+            text: 'Far far away, behind the word mountains, far from the countries Vokalia and Consonantia, there live the blind texts. Separated they live in Bookmarksgrove right at the coast of the Semantics, a large language ocean. A small river named Duden flows by their place. ',
+            path: picture,
+            author: 'John Doe',
+            modificationDate: '12/12/2020',
+        }
+    ];
```
Comment (upsu90) [inline web-client/src/pages/my-reviews.js]: can you please add more reviews? so we can see how it works

File: web-client/src/components/MyReviewsOneReview.jsx (hunk 3 above)
Comment (upsu90) [reply web-client/src/components/MyReviewsOneReview.jsx:36]: .... but it should look like this one: 
image
we forgot to change it in miro ;-;

**Review by upsu90 — CHANGES_REQUESTED** [review]

File: web-client/src/pages/my-reviews.js (hunk 4 above)
Comment (Kwolik) [reply web-client/src/pages/my-reviews.js]: Ok, no problem.

File: web-client/src/components/MyReviewsOneReview.jsx (hunk 3 above)
Comment (Kwolik) [reply web-client/src/components/MyReviewsOneReview.jsx:36]: I did now

**Review by JustD4nTe — APPROVED** [review]

**Review by upsu90 — APPROVED** [review]
great!

**Pull request merged**